inappmessaging/InAppMessagingExampleSwift/ViewController.swift

```diff
@@ -14,7 +14,10 @@
 //  limitations under the License.
 //
 
+import UIKit
+
 import Firebase
+import FirebaseAnalytics
```

Comment:
This one shouldn't be necessary because FirebaseAnalytics should come with the Firebase import.

Reply:
True... Let me take a closer look. I have an error locally when FirebaseAnalytics is not imported:
<img width="921" alt="Screen Shot 2021-08-05 at 10 22 09 AM" src="https://user-images.githubusercontent.com/1841926/128366383-f7008eb0-0149-4707-b7d8-4d2dc927915f.png">

Reply:
It seems to be related to flakes with Xcode module cache or something. I guess I had in a broken state. After removing DerivedData `Analytics` was marked as missing before the first build. I removed the import.